File: test/python/nose2-to-pytest/unconvertible/UNCONV-002.expected.py
# HAMLET-TODO [UNCONVERTIBLE-NOSE-PLUGIN]: nose2 plugin has no direct pytest equivalent
# Original: from nose2.tools.decorators import with_setup
# Manual action required: Find a pytest plugin or built-in feature that provides equivalent functionality
from nose2.tools.decorators import with_setup

def setup_func():
    pass

@with_setup(setup_func)
def test_with_setup():
    assert True

File: test/java/junit5-to-testng/unconvertible/UNCONV-002.expected.java
import org.testng.annotations.Test;

public class NestedTest {
    // HAMLET-TODO [UNCONVERTIBLE-NESTED]: JUnit 5 @Nested has no TestNG equivalent
    // Original: @Nested
    // Manual action required: Flatten nested test classes or use separate test classes
    @Nested
    class InnerTest {
        @Test
        public void testInner() {
            assert true;
        }
    }
}
